### PR TITLE
Add field weight to group.

### PR DESCRIPTION
**Pull request by reiterl**
Update tests, data. Write a migration and test.

### CODE REVIEW

File: global/data/example-data.json [hunk 1]
```diff
@@ -940,6 +940,7 @@
                 "projector.can_see",
                 "user.can_see"
             ],
+	    "weight": 1,
```
Comment (r-peschke) [inline global/data/example-data.json:943]: could you please set the indent to 4?

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: global/data/example-data.json (hunk 1 above)
Comment (reiterl) [reply global/data/example-data.json]: Updated the layout.

File: global/data/initial-data.json [hunk 2]
```diff
@@ -502,6 +502,7 @@
                 "projector.can_see",
                 "user.can_see"
             ],
+	    "weight": 1,
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/data/initial-data.json]: Updated the layout.

File: openslides_backend/action/actions/group/create.py [hunk 3]
```diff
@@ -22,10 +25,30 @@ class GroupCreate(CreateAction):
         ],
     )
     permission = Permissions.User.CAN_MANAGE
+    weight_map: Dict[int, int] = {}
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/group/create.py:28]: typed it in get_updated_instances.

[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/group/create.py:28]: I simplified the set_weight method, so it no longer need a weight_map.

File: openslides_backend/action/actions/group/create.py [hunk 4]
```diff
@@ -22,10 +25,30 @@ class GroupCreate(CreateAction):
...
             instance["permissions"] = filter_surplus_permissions(
                 instance["permissions"]
             )
+        self.set_weight(instance)
         return instance
+
+    def set_weight(self, instance: Dict[str, Any]) -> None:
+        meeting_id = instance["meeting_id"]
+        if meeting_id in self.weight_map:
+            max_weight = self.weight_map[meeting_id]
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/group/create.py:46]: Simplified the code.

File: openslides_backend/action/actions/group/create.py [hunk 5]
```diff
@@ -22,10 +25,30 @@ class GroupCreate(CreateAction):
...
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         if instance.get("permissions"):
             instance["permissions"] = filter_surplus_permissions(
                 instance["permissions"]
             )
+        self.set_weight(instance)
         return instance
+
+    def set_weight(self, instance: Dict[str, Any]) -> None:
+        meeting_id = instance["meeting_id"]
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/group/create.py:44]: Simplified the code.

File: tests/system/action/meeting/test_clone.py [hunk 6]
```diff
@@ -46,6 +46,7 @@ def setUp(self) -> None:
             "group/1": {
                 "meeting_id": 1,
                 "name": "testgroup",
+                "weight": 1,
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/action/meeting/test_clone.py:49]: Add a test.

File: openslides_backend/action/actions/group/create.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/group/create.py:45]: Add two tests to check the weight in multi group creation.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**